src/generators/src/packagejson.js

```diff
@@ -54,7 +54,8 @@ module.exports =
         '"nodemon": "^2.0.19"',
         '"mocha": "^10.0.0"',
         '"lodash.camelcase": "^4.3.0"',
-        '"sugar-env": "^1.5.14"'
+        '"sugar-env": "^1.5.14"',
+        '"uuid": "^9.0.0"'
```

Comment:
Can we use the native uuid?
https://nodejs.org/api/crypto.html#cryptorandomuuidoptions

Reply:
@dalssoft I updated the code to use the native `crypto` module from Node.js to generate UUIDs, as suggested. Thank you for the suggestion!